src/Form/MiradorConfigForm.php

```diff
@@ -23,6 +30,30 @@ public function getFormId() {
   public function buildForm(array $form, FormStateInterface $form_state) {
     $form = parent::buildForm($form, $form_state);
     $config = $this->config('islandora_mirador.settings');
+    $form['mirador_library_fieldset'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Mirador library location'),
+    ];
+    $form['mirador_library_fieldset']['mirador_library_installation_type'] = [
+      '#type' => 'radios',
+      '#options' => [
+        'local'=> $this->t('Local library placed in /libraries inside your webroot.'),
+        'remote' => $this->t('Default remote location ='),
```

Comment:
```suggestion
        'remote' => $this->t('Default remote location'),
```

... unless we wanted to lookup/include the value in the string?